.cookiecutter/includes/tox/setenv

```diff
@@ -7,7 +7,6 @@ H_CA_DATABASE_URL = {env:H_CA_DATABASE_URL:postgresql://report-fdw:password@h_po
 LMS_US_DATABASE_URL = {env:LMS_US_DATABASE_URL:postgresql://report-fdw:password@lms_postgres_1:5432/postgres}
 LMS_CA_DATABASE_URL = {env:LMS_CA_DATABASE_URL:postgresql://report-fdw:password@lms_postgres_1:5432/postgres}
 # Used when testing tasks via `tox`:
-dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:5436/postgres}
```

Comment:
No need for report to put this in the `setenv` include anymore: the cookiecutter does this for you now